### PR TITLE
PokeApi JS Wrapper configuration and fetch refactoring

### LINKED ISSUE #74: Refactor Pokemon API Calls with Wrapper
**Is your feature request related to a problem? Please describe.**
Wrapper not configured properly.

**Describe the solution you'd like**
Refactor wrapper and fetching.

### PR DESCRIPTION
**Pull request by connielion**
## Changes
1. Save `pokeapi-js-wrapper` as dev dependency.
2. Refactored fetch methods.
3. Added custom options for the wrapper.
4. Removed unneeded script in index.html to get rid of reference error in console.
5. Added comments for doc.
6. Conditional check for `moves`.

## Purpose
JS wrapper was not installed into project and was not configured properly. 

## Approach
Install as dev dependency and initialized `Pokedex` with options for caching passed in.

Closes #74

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "@testing-library/user-event": "^12.8.3",
     "chart.js": "^3.5.0",
     "node-sass": "^6.0.1",
+    "pokeapi-js-wrapper": "^1.2.0",
```
Comment (BlackCubes) [inline package.json:14]: What's dis 👀

**Review by kmausolf — COMMENTED** [review]
Nice refactor!  Is there a reason we removed the service worker?  We need that in order to cache images.

File: src/components/PokeCard.js [hunk 2]
```diff
@@ -38,9 +38,11 @@ const PokeCard = ({
...
   const getMovesByPokemon = async (id) => {
     const pokemon = await fetchPokemon(id);
-    const moves = pokemon.moves;
-    const someMoves = await getMoveset(moves, 4);
-    setMoveSet(someMoves);
+    if (pokemon.moves) {
+      const moves = pokemon.moves;
+      const someMoves = await getMoveset(moves, 4);
+      setMoveSet(someMoves);
+    }
```
Comment (BlackCubes) [inline src/components/PokeCard.js:45]: Interesting

**Comment by connielion** [comment]
> Nice refactor! Is there a reason we removed the service worker? We need that in order to cache images.

Oops! Forgot to put it back into the project. Will fix that now.

**Review by BlackCubes — APPROVED** [review]
Everything looks good and clean!

File: package.json (hunk 1 above)
Comment (connielion) [reply package.json:14]: Dr. Google~

**Review by kmausolf — APPROVED** [review]
Looking good!  Keeping that pivot game strong!

**Review by JorgeLVilla — COMMENTED** [review]
Great job with the comments! Makes it easier to read.

**Review by BlackCubes — COMMENTED** [review]
Nice changes!

**Pull request merged**